com.unity.render-pipelines.universal/Documentation~/features/rp-converter.md

```diff
@@ -0,0 +1,99 @@
+
+## How to use the Render Pipeline Converter
+
+To convert project assets:
+
+1. Select **Window** > **Rendering** > **Render Pipeline Converter**. Unity opens the Render Pipeline Converter window.
+
+    ![Render Pipeline Converter dialog](../Images/rp-converter/rp-converter-dialog.png)
+
+2. Select the conversion type.
+
+    ![Conversion type](../Images/rp-converter/conversion-types.png)
+
+3. Depending on the conversion type, the dialog shows the available converters. Select or clear the check boxes next to converter names to enable or disable the converters.
+
+    ![Select converters](../Images/rp-converter/select-converters.png)
+
+    For the list of available converters, see the section [Converters](#converters).
+
+4. Click **Initialize Converters**. The Render Pipeline Converter preprocesses the assets in the project and shows the list of elements to convert. Select or clear check boxes next to assets to include or exclude them from the conversion process.
+
+    ![Initialize converters](../Images/rp-converter/initialize.png)
+
+    **Yellow icon**: a yellow icon next to an element indicates that a user action might be required to run the conversion. Hover the mouse pointer over the icon to see the description of the issue.
+
+5. Click **Convert Assets** to start the conversion process.
+
+    > **NOTE:** The conversion process makes irreversible changes to the project. Back up your project before the conversion.
+
+    When the converter finishes processing all the selected elements, it shows the status of each element in the window.
+
+    ![Conversion finished](../Images/rp-converter/conversion-finished.png)
+
+    **Green check mark**: the conversion went without issues.
+
+    **Red icon**: the conversion failed.
+
+6. With the converter window open, inspect the elements with the warnings. After reviewing the converted project, close the Render Pipeline Converter window.
+
+## <a name="converters"></a>Conversion types and converters
+
+The Render Pipeline Converter let's you select one of the following conversion types:
+
+* Built-in Render Pipeline 2D to URP 2D
+
+* Upgrade 2D URP Assets
+
+* Built-in Render Pipeline to URP
+
+When you select on of the conversion types, the tool shows you the available converters.
+
+The following sections describe the converters available for each conversion type.
+
+### Built-in Render Pipeline 2D to URP 2D
+
+This conversion type converts elements of a project from Built-in Render Pipeline 2D to URP 2D.
+
+Available converters:
+
+* **Material and Material Reference Upgrade**
+
+    This converter converts all Materials and Material references from Built-in Render Pipeline 2D to URP 2D.
+
+### Upgrade 2D URP Assets
+
+This conversion type upgrades assets of a 2D project from an earlier URP version to the current URP version.
+
+Available converters:
+
+* **Parametric to Freeform Light Upgrade**
+
+    This converter converts all parametric lights to freeform lights.
```

Comment:
@martint-unity @unity-cchu question, should URP to URP upgrade be handled in this converter?

Reply:
URP to URP is still a conversion, so it makes sense. With all this tooling centralised it makes it more predictable for users in the future to know where to find upgraders and converters in the future. 

we built this framework in a way that is flexable enough to encompass things like upgrades, conversions and even maybe setting up a scene.